### PR TITLE
Updated the Polyglot Do it yourself section

### PR DESCRIPTION
**Pull request by shraddhavp**
this PR updated the Polyglot DIY section ,here have deleted sections that already present in Python wechaty section https://wechaty.js.org/docs/polyglot/python/ and also added the new polyglot architecture image

### CODE REVIEW

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Review by Soumi7 — APPROVED** [review]
LGTM

**Review by proudofsimin — APPROVED** [review]

File: docusaurus/docs/polyglot/diy/overview.md [hunk 1]
```diff
@@ -5,54 +5,14 @@ title: 'Polyglot: DIY'
...
-  +-------------------------------------------------------+
-  |                Wechaty Puppet Abstract                |
-  |                                                       |
-  |                   (wechaty-puppet)                    |
-  +-------------------------------------------------------+
-
-  +--------------------------+ +--------------------------+
-  |      Pad Protocol        | |      Web Protocol        |
-  |                          | |                          |
-  | wechaty-puppet-padlocal  | |  (wechaty-puppet-wechat) |
```
Comment (huan) [inline docusaurus/docs/polyglot/diy/overview.md:46]: Please keep the asciiart image, which is easier to be maintained.

File: docusaurus/docs/polyglot/diy/overview.md [hunk 2]
```diff
@@ -5,54 +5,14 @@ title: 'Polyglot: DIY'
...
 
-Create your own Wechaty Puppet Provider
-
-:::
-
-We built Wechaty in TypeScript in the early days, it is not hard to translate the TypeScript(TS) to Python, Go, and Java languages because [wechaty](https://github.com/wechaty/wechaty) has only 3,000 lines of the TS code, they are well designed and de-coupled by the [wechaty-puppet](https://github.com/wechaty/wechaty-puppet/) abstraction. So after we have translated those 3,000 lines of TypeScript code, we will almost be done.
-
-As we have already an ecosystem of Wechaty in TypeScript, so we will not have to implement everything in other languages, especially, in Feb 2020, we have finished the [wechaty_grpc](https://github.com/wechaty/grpc) service abstracting module with the [wechaty-puppet-service](https://github.com/wechaty/wechaty-puppet-service) implementation.
```
Comment (huan) [inline docusaurus/docs/polyglot/diy/overview.md:16]: rename `wechaty_grpc` to `wechaty-grpc`, which is our correct NPM name.

File: docusaurus/docs/polyglot/diy/overview.md [hunk 3]
```diff
@@ -5,54 +5,14 @@ title: 'Polyglot: DIY'
...
-
-  +--------------------------+ +--------------------------+
-  |      Pad Protocol        | |      Web Protocol        |
-  |                          | |                          |
-  | wechaty-puppet-padlocal  | |  (wechaty-puppet-wechat) |
-  +--------------------------+ +--------------------------+
-  +--------------------------+ +--------------------------+
-  |     Friday Protocol      | |       Mac Protocol       |
-  |                          | |                          |
-  |  (wechaty-puppet-frida)  | |   (wechaty-puppet-mac)   |
```
Comment (huan) [inline docusaurus/docs/polyglot/diy/overview.md:51]: Please help us to update the diagram by the following changes:

1. `Friday Protocol` -> `Windows Protocol`
1. `wechaty-puppet-frida` -> `wechaty-puppet-xp`
2. `Mac Protocol` -> `Whatsapp Protocol`
3. `wechaty-puppet-mac` -> `wechaty-puppet-whatsapp`

**Review by huan — DISMISSED** [review]
Thanks for the update!

Please follow my reviews and I believe this PR will be ready to go after we have finished them!

File: docusaurus/docs/polyglot/diy/overview.md (hunk 2 above)
Comment (shraddhavp) [reply docusaurus/docs/polyglot/diy/overview.md:16]: @huan done

File: docusaurus/docs/polyglot/diy/overview.md (hunk 1 above)
Comment (shraddhavp) [reply docusaurus/docs/polyglot/diy/overview.md:46]: done

File: docusaurus/docs/polyglot/diy/overview.md (hunk 3 above)
Comment (shraddhavp) [reply docusaurus/docs/polyglot/diy/overview.md:51]: done @huan

**Comment by shraddhavp** [comment]
> Thanks for the update!
> 
> Please follow my reviews and I believe this PR will be ready to go after we have finished them!

@huan done with the changes as suggested ,thanks for the suggestion

**Pull request merged**